CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Next Release] - Release Date
 ### Added
 - [server] Add --debug flag. For now only print the stack trace on panic/recover.
+- [client] Add --context flag. To use a cluster different of current-cluster.
```

Comment:
Isn't a problem in your PR but this entry already in `Added` section of the changelog, for this reason, that word `add` is unnecessary.